tests/Moq.Tests/ProtectedAsMockFixture.cs

```diff
@@ -298,6 +381,10 @@ public void VerifyGet_includes_failure_message_in_exception()
 			Assert.Contains("Was not queried.", exception.Message);
 		}
 
+		public interface INested
+		{
+			int Value { get; set; }
+		}
 		public abstract class Foo
```

Comment:
Detail: for consistency with existing formatting, please add a blank line between those two lines of code.